users/serializers.py

```diff
@@ -45,3 +46,13 @@ def create(self, validated_data):
         instance.is_active = True
         instance.save(update_fields=['is_active'])
         return instance
+
+
+class UserDetailSerializer(serializers.ModelSerializer):
+    owned_projects = serializers.StringRelatedField(many=True)
+    co_projects = serializers.StringRelatedField(many=True)
+
+    class Meta:
+        model = get_user_model()
+        fields = ['pk', 'email', 'last_login', 'is_active',
```

Comment:
last_login, is_active its useful information? for example for frontend?

Reply:
Depends on frontend's needs ;)